mass_operation_abstract/i18n/de.po

```diff
@@ -0,0 +1,189 @@
+# Translation of Odoo Server.
```

Comment:
Ideally translations should be done using transifex.

Reply:
Sorry. I did not know about that. I see OCA also have a tool called weblate. In future situations Ill do it that way. Thx for your review :smile:.

Reply:
Exactly, weblate is the new tool! Thanks for contributing :)